src/services/services.tsx

```diff
@@ -0,0 +1,12 @@
+const backend_URL: string = 'https://lessoner.herokuapp.com/en';
+
+const isEmailExists = async (email: string): Promise<boolean> => {
+    const res = await fetch(`${backend_URL}/check_email?email=${email}`);
+
+    if (!res.ok) {
+        throw new Error(`Error code ${res.status}`);
+    }
+    return (await res.json()).email_exists
```

Comment:
let's not use that pattern: `(await func()).prop`

please, split that into 2 lines:

```
const data = await res.json()
return data.email_exists
```

reason - will be easier to add `console.log` or `debugger` to check returned data

Reply:
done